### PR TITLE
yys/step1

### PR DESCRIPTION
**Pull request by AlvinYou**

### CODE REVIEW

**Review by cjy9306 — DISMISSED** [review]
all

**Review by korJiusngPark — DISMISSED** [review]
good

File: 1/7_11718_yys.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+var fs = require('fs');
+var input = fs.readFileSync('/dev/stdin').toString().split('\n');
+
+for(var i in input){
+  console.log(input[i])
+}
```
Comment (jukqaz) [inline 1/7_11718_yys.js]: Does the code pass without trim in JS?

**Review by kor-haru — CHANGES_REQUESTED** [review]
Keep the source code consistent plz. especially semicolon

**Review by jukqaz — DISMISSED** [review]

**Comment by AlvinYou** [comment]
@dopingcat ok. cleared semicolon of all files.

**Pull request merged**